### ISSUE TITLE
RECEIVE_BOOT_COMPLETED permission removed

### ISSUE THREAD

**Issue by Abdelrhmanwalid** [Buglife/buglife-android#61]
In the latest version 1.5.0 `RECEIVE_BOOT_COMPLETED` permission was removed 
I think that it's still needed for jobs to persisted 
https://github.com/Buglife/buglife-android/blob/c4a681fe6703481325002ef38825d1f15b448b13/src/main/java/com/buglife/sdk/BugReporterImpl.java#L128

>setPersisted
>Set whether or not to persist this job across device reboots. 
>Requires Manifest.permission.RECEIVE_BOOT_COMPLETED
https://developer.android.com/reference/android/app/job/JobInfo.Builder.html#setPersisted(boolean)

without the permission this exception is thrown on submitting the report, manually adding it to the app fixes the issue
```
    java.lang.IllegalArgumentException: Error: requested job be persisted without holding RECEIVE_BOOT_COMPLETED permission.
        at android.os.Parcel.createException(Parcel.java:1954)
        at android.os.Parcel.readException(Parcel.java:1918)
        at android.os.Parcel.readException(Parcel.java:1868)
        at android.app.job.IJobScheduler$Stub$Proxy.schedule(IJobScheduler.java:184)
        at android.app.JobSchedulerImpl.schedule(JobSchedulerImpl.java:44)
        at com.buglife.sdk.BugReporterImpl.reportWithJobScheduler(BugReporterImpl.java:132)
        at com.buglife.sdk.BugReporterImpl.report(BugReporterImpl.java:97)
        at com.buglife.sdk.Client.submitReport(Client.java:307)
        at com.buglife.sdk.Buglife.submitReport(Buglife.java:253)
        at com.buglife.sdk.ReportActivity.submitReport(ReportActivity.java:232)
        at com.buglife.sdk.ReportActivity.onOptionsItemSelected(ReportActivity.java:168)
        at android.app.Activity.onMenuItemSelected(Activity.java:3562)
        at androidx.fragment.app.FragmentActivity.onMenuItemSelected(FragmentActivity.java:436)
        at androidx.appcompat.app.AppCompatActivity.onMenuItemSelected(AppCompatActivity.java:196)
        at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109)
        at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109)
        at androidx.appcompat.app.ToolbarActionBar$2.onMenuItemClick(ToolbarActionBar.java:64)
        at androidx.appcompat.widget.Toolbar$1.onMenuItemClick(Toolbar.java:204)
        at androidx.appcompat.widget.ActionMenuView$MenuBuilderCallback.onMenuItemSelected(ActionMenuView.java:781)
        at androidx.appcompat.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:840)
        at androidx.appcompat.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:158)
        at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:991)
        at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:981)
        at androidx.appcompat.widget.ActionMenuView.invokeItem(ActionMenuView.java:625)
        at androidx.appcompat.view.menu.ActionMenuItemView.onClick(ActionMenuItemView.java:151)
        at android.view.View.performClick(View.java:6671)
        at android.view.View.performClickInternal(View.java:6640)
        at android.view.View.access$3100(View.java:791)
        at android.view.View$PerformClick.run(View.java:26173)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6940)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: android.os.RemoteException: Remote stack trace:
        at com.android.server.job.JobSchedulerService$JobSchedulerStub.schedule(JobSchedulerService.java:2592)
        at android.app.job.IJobScheduler$Stub.onTransact(IJobScheduler.java:60)
        at android.os.Binder.execTransact(Binder.java:752)
```

**Comment by danhd123** [comment]
Fixed in d6b60e1

**Comment by danhd123** [comment]
Thanks for the report, @Abdelrhmanwalid. This is fixed in 1.5.1. Apologies for the inconvenience.
